.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,14 @@
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
using this means that tests will not run on the release tag, which is probably fine, but it also means tests won't run on the PRs that release-please opens, you can to copy this https://github.com/contiamo/go-base/blob/8b3392348c21a0f991fd7e820f2f7bfcc8681c51/.github/workflows/ci.yaml#L91-L94